The parameter to use in the probe is -PickHostnameFromBackendHttpSett…

surajmb opened the pull request:
…ings

The parameter to use in the probe is -PickHostnameFromBackendHttpSettings, it was wrongly mentioned as -PickHostnameFromBackendAddress, which is a different configuration to be made on the HTTP Settings.

azurecla commented:
Hi __@jastru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

surajmb committed:
The parameter to use in the probe is -PickHostnameFromBackendHttpSettings

The parameter to use in the probe is -PickHostnameFromBackendHttpSettings, it was wrongly mentioned as -PickHostnameFromBackendAddress, which is a different configuration to be made on the HTTP Settings.
